### PR TITLE
Issue 64

### PR DESCRIPTION
**Pull request by jaffa80**
#64 Updating nav elements to include differentiating aria-labels for improved accessibility

### CODE REVIEW

File: src/_includes/partials/global/site-foot.njk [hunk 1]
```diff
@@ -10,7 +10,8 @@
         </div>
       {% endif %}
       <div>
-        {% include "partials/components/nav.njk" %}
+      {% set ariaLabel = 'footer navigation' %}
+      {% include "partials/components/nav.njk" %}
```
Comment (hankchizljaw) [inline src/_includes/partials/global/site-foot.njk]: I know it's super picky, but can we indent these in please?

Comment (jaffa80) [reply src/_includes/partials/global/site-foot.njk]: No worries at all! I guess prettier cannot format njk files. I struggle with a few other extensions also but they messed up the formatting completely. Do you just do the formatting manually?

Comment (jaffa80) [reply src/_includes/partials/global/site-foot.njk]: OK.. trying that again... some formatting went crazy. Let me know if this is ok now. Thanks for your help and patience!

Comment (hankchizljaw) [reply src/_includes/partials/global/site-foot.njk]: Most stuff gets caught by prettier, but I think nunjucks files get ignored. Thanks, I'll merge this in!

**Pull request merged**